SwanCustomEnvironments/swancustomenvironments/scripts/makenv.sh

```diff
@@ -226,6 +226,15 @@ if [ ! -d "${GIT_REPO_PATH}" ]; then
     mv ${TMP_REPO_PATH} ${GIT_HOME}
 fi
 
+# Define the global gitignore file location
```

Comment:
General comment: this git ignore feature should be implemented elsewhere, possibly in some layer of the user image, and it should not be tied to custom environments. The same issue can happen if we are using JupyterLab with LCG and the git extension -- i.e. the git extension is not tied exclusively to the custom environments.